src/delta.c

```diff
@@ -110,7 +109,12 @@ void __create_delta(struct file *file, int from_version)
 		goto out;
 	}
 
-	ret = system(sanitycheck);
+	string_or_die(&param1, "%s", newfile);
+	string_or_die(&param2, "%s", testnewfile);
+	char *const sanitycheck[] = { "cmp", "-s", param1, param2, NULL};
+	ret = system_argv(sanitycheck);
+	free(param1);
+	free(param2);
 	if (ret == -1 || !WIFEXITED(ret) || WEXITSTATUS(ret) == 2) {
```

Comment:
system_argv() already calls WIFEXITED and WEXITSTATUS, so this line needs updating.